src/main/java/mate/academy/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.bookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(
+        @NotEmpty
+        @Length(min = 8,max = 35)
+        @Email
```

Comment:
It's better to create custom Email annotation and validator for it. 
Read, what exactly check `@Email`

Reply:
I'm sorry I didn't understand what I should do